ExpensesTracker/Forms/MainWindow.cs

```diff
@@ -27,15 +27,32 @@ public MainWindow()
         private void MainWindow_Load(object sender, EventArgs e)
         {
             _accounts = _storage.GetAccounts();
-            _accountOperations = _storage.GetAccountOperations();
+
+            CreateDefaultAccount();
 
             accountBox.DataSource = null;
             accountBox.DataSource = _accounts;
             accountBox.DisplayMember = "Name";
-            accountBox.SelectedIndex = 0;
 
             operationsTable.AutoGenerateColumns = false;
+
+            accountBox.SelectedIndex = 0;
```

Comment:
Subscribe method on SelectedIndexChanged event after you set SelectedIndex as "0". Call ShowTable and ShowBalance method explicitly.

Reply:
You don't need to unsubscribe, do an operation, and then subscribe again. 
Just remove a subscription from MainWindows.Designer and subscribe after the operation.